Gruntfile.js

```diff
@@ -292,6 +253,14 @@ var _              = require('lodash'),
                     }
                 },
 
+                jshintClient: {
+                    command: 'cd core/client && grunt jshint'
+                },
+
+                jscsClient: {
+                    command: 'cd core/client && grunt jscs'
```

Comment:
👍  Can you finish your PR?Then i can set mine to `[WIP]` and wait until your's is merged.